.github/workflows/statuses.yml

```
@@ -0,0 +1,32 @@
+name: Update Statuses Board
+
+on:
+  push:
+    branches:
+      - main
+    # Only run workflow on changes to component docs
+    paths:
+      - 'docs/content/**'
+
+jobs:
+  build-docs:
+    name: Update statuses project
```

Should we rename this file `status.yml` to be consistent with the status page title ([PR](https://github.com/primer/react/pull/1669))?

From @yaili:

> can we rename the page to “Component status” - I think the plural of status can be both, and it looks nicer 😄





```suggestion
    name: Update status project
```